src/webapi/mutation_observer.rs

```diff
@@ -161,6 +161,7 @@ impl MutationObserver {
 ///
 /// When the `MutationObserverHandle` is dropped, the [`disconnect`](#method.disconnect)
 /// method will automatically be called.
+// TODO derive PartialEq, Eq, etc. ?
```

Comment:
This can compare the `callback_reference`.

Reply:
Like I said above, the question isn't *how* to implement it, it's whether to implement it *at all*.

Reply:
I don't see why not?

Reply:
It's a commitment to the public API. If we later change the internal implementation it might change the behavior (or even make comparing for equality impossible).

Implementing `PartialEq` for `ReferenceType` is fine, because its behavior is stable, but the Handle types are different.

Reply:
Since *we* need a stable handle inside of this I think this always should be comparable?

But if you'd rather be conservative on this I don't mind. (I don't have strong feelings either way.)